### PR TITLE
Fixes #9247 feat(nimbus): updated UI to support rendering detailed changes

### LINKED ISSUE #9247: [UI] Update the history page  to render changes based on event type
Follow up to #9055 

**Current approach**
- render the more detailed changes  inside a modal 
- display the data in a proper format

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3750)

### PR DESCRIPTION
**Pull request by Avi-88**
Dependant on #9246 
Because

- the changes of type `list` or `json` weren't being displayed properly due to large size

This commit

- updates the formatting of such fields and renders them into a split display

**Screenrecording**

https://drive.google.com/file/d/1pCFcTYTA-KwdiqvjPfUsp0npMSGXHAiC/view

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -76,7 +76,7 @@ target/
 **/.vscode/*
 
 # Hosted assets
-experimenter/experimenter/static/scripts/
+experimenter/experimenter/static/scripts/bundle.js
```
Comment (yashikakhurana) [inline .gitignore:79]: why this change?

File: experimenter/experimenter/experiments/changelog_utils.py [hunk 2]
```diff
@@ -87,3 +92,66 @@ def generate_nimbus_changelog(experiment, changed_by, message, changed_on=None):
...
+            new_value = json.dumps(new_value, indent=2)
+
+    if isinstance(field_instance, models.BooleanField):
+        event_name = ChangeEventType.BOOLEAN.name
+
+    if field_name == "status" or field_name == "publish_status":
+        event_name = ChangeEventType.STATE.name
+
+    if event_name == ChangeEventType.DETAILED.name:
+        change_message = f"{changelog.changed_by} changed value of {field_display_name}"
```
Comment (yashikakhurana) [inline experimenter/experimenter/experiments/changelog_utils.py:139]: same as other PR, to use `elif`

File: experimenter/experimenter/templates/components/change_card.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
+<div x-data="{expand:false}" :class="{'bg-white border border-gray-100 shadow-md rounded-md': expand }"  class="p-2">
```
Comment (yashikakhurana) [inline experimenter/experimenter/templates/components/change_card.html:1]: I believe one is for desktop and one is for mobile version, would suggest adding a comment to keep the them separate What css classes have been applied

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
few minor suggestions :)

File: .gitignore (hunk 1 above)
Comment (Avi-88) [reply .gitignore:79]: Oh , this was for the modal script I'll update it

File: experimenter/experimenter/templates/components/change_card.html (hunk 3 above)
Comment (Avi-88) [reply experimenter/experimenter/templates/components/change_card.html:1]: This was for the expanded view of the change, the padding remains the same but the rest of the styling is added when the card expands, I'll add a comments that specify behaviour if dependant on alpine states

**Review by yashikakhurana — APPROVED** [review]
thank you for adding comments too, neat @Avi-88 👏 👏
